c++/src/kj/compat/http.c++

```diff
@@ -3901,6 +3901,23 @@ public:
     return transferredBytes;
   }
 
+  kj::Maybe<kj::String> getPreferredExtensions(ExtensionsContext ctx) override {
```

Comment:
I believe this method will never be called, since the wrapper will always use the `destinationPumping{To,From}` directly. So the body of this can be `KJ_UNREACHABLE`. Same goes for all the specific state types I think.